src/patient.js

```diff
@@ -68,12 +68,14 @@ export default class Patient {
   }
 
   toSuggest() {
+    return this.toDiagnosis();
```

Comment:
In v3 `toSuggest();` can basically be a proxy for `toDiagnosis();`
because we evidence in suggest now, sorted out what we discussed on Slack, now the app handles suggested symptoms:
![Screenshot 2021-01-21 at 09 28 53](https://user-images.githubusercontent.com/39158460/105324311-8bfc7980-5bcb-11eb-9a5a-fc080b374381.png)